tests/integration-tests/tests/update/test_update.py

```diff
@@ -48,20 +49,33 @@ def test_update_slurm(region, pcluster_config_reader, s3_bucket_factory, cluster
     # Create S3 bucket for pre/post install scripts
     bucket_name = s3_bucket_factory()
     bucket = boto3.resource("s3", region_name=region).Bucket(bucket_name)
-    for script in ["preinstall.sh", "postinstall.sh", "updated_preinstall.sh", "updated_postinstall.sh"]:
+    for script in [
+        "preinstall.sh",
+        "postinstall.sh",
+        "updated_preinstall.sh",
+        "updated_postinstall.sh",
+        "postupdate.sh",
+        "updated_postupdate.sh",
+    ]:
         bucket.upload_file(str(test_datadir / script), f"scripts/{script}")
 
     # Create cluster with initial configuration
     init_config_file = pcluster_config_reader(resource_bucket=bucket_name, bucket=bucket_name)
     cluster = clusters_factory(init_config_file)
 
+    # Check update hook is NOT executed at cluster creation time
+    assert_that(os.path.exists("/tmp/postupdate_out.txt")).is_false()
```

Comment:
Non-blocking: for assertions like this, rather than adding a comment saying what we are checking, it might be better to create a method that says what the test is doing (i.e. `_check_update_hook_has_not_run`).

Reply:
I agree in general,
in this case I'd say we can keep it as is, to avoid to introduce a one-liner function.

Reply:
Imho nothing wrong with a one-liner function if it helps make the code self-documenting (my thinking is that I prefer to keep comments for explaining stuff like why an approach was chosen over another, rather than what the code does), in any case it's not blocking, for me the PR can be merged as soon as we have merged the CLI patch and run the test.

Reply:
We can discuss about one-liner functions: https://mytwoocents.com/2017/03/25/one-line-methods-can-easily-become-an-antipattern/ ;)